azure/packages/azure-client/.eslintrc.js

```diff
@@ -31,6 +31,21 @@ module.exports = {
         // Ensure jsdoc/tsdoc comments contain a main description component
         // (disallows empty comments / only tags).
         "jsdoc/require-description": ["error", { checkConstructors: false }],
+
+        // TODO: remove once dependency on base config has been updated.
+        "@typescript-eslint/explicit-member-accessibility": [
```

Comment:
It's unclear to me that checking this in is worth while. Doing so just to remove it in a few days seems low value.

Might be better to test with npm-link to find cases with violate the version of the lint config in the repo.

Reply:
That's what these changes represent. They are pre-fixing to accommodate the new rule, but that new rule violates the existing rules, so unless we change the local configs to match (for now), then linting will fail.

Reply:
so the current default bans explicit constructor public scope instead of just not requiring it?

Reply:
Correct.